src/Policies/RolePolicy.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace TCG\Voyager\Policies;
+
+use TCG\Voyager\Contracts\User;
+
+class RolePolicy extends BasePolicy
+{
+    public function read(User $user, $model)
+    {
+        $role_greater = $user->role->order == 1 || $model->order > $user->role->order;
```

Comment:
What's the guarantee that `admin` has order=1? I don't like magic numbers like this.

Reply:
@fletch3555 
In the fact I do that to `Developer` role.
I created many projects depends on `voyager` in every time I need `developer` role to prevent admin to edit in `BREAD` tables